frontends/web/src/routes/buy/exchange.tsx

```diff
@@ -92,12 +94,12 @@ export const Exchange = ({ code, accounts }: TProps) => {
       return;
     }
 
-    const userRegion = getRegionNameFromLocale(i18n.language);
+    const userRegion = getRegionNameFromLocale(nativeLocale || '');
     //Region is available in the list
     const regionAvailable = !!(regionList.regions.find(region => region.code === userRegion));
     //Pre-selecting the region
     setSelectedRegion(regionAvailable ? userRegion : '');
-  }, [regionList, config]);
+  }, [regionList, config, nativeLocale]);
```

Comment:
Bit of explanation:

`userRegion` will return empty string if:
1. nativeLocale is empty string
2. nativeLocale is undefined
3. nativeLocale is invalid string / locale e.g `asd` or `en-XX`.

When `userRegion` is empty, `regionAvailable` will be `false`. When it's `false`, `selectedRegion` will be empty string -> which means it'll select the placeholder "Select Region" selection.

How did it select that placeholder value when it's empty string? It's due to the code below

```
  <Select
    className={style.extraLeftSpace}
    options={[{
      text: t('buy.exchange.selectRegion'),
      disabled: true,
      value: '',
    },
    ...regions]
    }
    value={selectedRegion}
    onChange={handleChangeRegion}
    id="exchangeRegions"
  />
```